cleverhans/utils.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import unicode_literals
 
 import numpy as np
+import tensorflow as tf
```

Comment:
At the moment, we should refrain from importing tensorflow at the top of files (because although theano is deprecated, we are still supporting for a few months --- and that's probably why the Travis tests failed on the Theano setup)